tools/cldr-rdf/external/entityToCode.tsv

```diff
@@ -1669,7 +1669,7 @@ http://www.wikidata.org/entity/Q23014	pfl
 http://www.wikidata.org/entity/Q23526	sla
 http://www.wikidata.org/entity/Q23855	auq
 http://www.wikidata.org/entity/Q25164	nn
-http://www.wikidata.org/entity/Q25167	no
+http://www.wikidata.org/entity/Q25167	nb
```

Comment:
This is a generated file. Will have to watch it when its next generated.

Reply:
This is a snapshot for review purposes (an output, not an input).  The actual content is regenerated in the ST.  So this may not work properly and may need some followon. Shouldn't hold this PR though. s

Reply:
I had looked up what that Wikidata entity referred to and it is in fact nb, not no. I had changed it before when we aliasing so no really meant nb.

Reply:
@pedberg-icu right, but nothing _reads_ entityToCode.tsv, it's output to give a historical record of the process.